tools/install_morfessor.sh

```diff
@@ -0,0 +1 @@
+extras/install_morfessor.sh
```

Comment:
I think you don't have to add this symlink (that is done only for historical reasons for some packages)

Reply:
okay got it. I deleted the symlink